prime-dotnet-webapi/Migrations/20230609220819_AddSocialWorkers.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AddSocialWorkers : Migration
```

Comment:
Somehow this data migration snuck into this branch?

Reply:
I added the social worker migration into this branch so the ID of the license class will align properly. The only drawback is the social worker change need to go first. 
However, I noticed the migration for Oral Health Professional is not executed in PR. I am currently investigating that.